src/styles/main.scss

```
@@ -10,5 +14,20 @@ body {
   box-sizing: border-box;
 }
 
-@import "_header.scss";
-@import "_catalog.scss";
+.catalog__card {
+  transition-duration: $transition-duration;
+}
+
+.nav__link {
+  transition-duration: $transition-duration;
+}
+
+.catalog__card:hover {
+  transform: translateY(#{-$card-transform-distance});
```

I think we need to use scale(1,2) instead of translateY
```suggestion
  transform: scale(1,2);
```

---

Yeah, I just misread the instruction, thanks